pkgs/development/libraries/lmdb/default.nix

```diff
@@ -18,8 +18,12 @@ stdenv.mkDerivation rec {
 
   outputs = [ "bin" "out" "dev" ];
 
-  makeFlags = [ "prefix=$(out)" "CC=cc" ]
```

Comment:
This would also work: 

```
makeFlags = [
  "prefix=$(out)" 
  "CC=${stdenv.cc.targetPrefix}cc"
  "AR=${stdenv.cc.targetPrefix}ar"
]
```

Reply:
Yes, that's certainly nicer :ballot_box_with_check: